GroupProject/PlayerInfoSchema.json

```diff
@@ -21,10 +21,14 @@
           "type": "integer"
         },
         "PlayerName": {
-          "type": "string"
+          "type": "string",
```

Comment:
Added length validation for PlayerName. Although validation does not seem to be integrated into Index page yet, this will validate the string has a length between the values below.

Source: https://json-schema.org/understanding-json-schema/reference/string.html